plugins/azure_sentinel/help.md

```diff
@@ -6,44 +6,1322 @@ Azure Sentinel is Microsoft's' automated security service.
+|client_secret|credential_secret_key|None|True|The application secret that you generated for your app in the app registration portal|None|ef50c6bx9umaik9agvoxtoqec2fg9f0y|
+|tenant_id|string|None|True|The Azure Tenant ID is a Global Unique Identifier (GUID) for your Azure Active Directory Tenant|None|5ceea899-ae8c-4ff1-fffe-353646eeeff0|
+
+Example input:
+
+```
+{
+  "client_id": "c163eff0-d1a1-4618-ee2a-453534f43cee",
+  "client_secret": "ef50c6bx9umaik9agvoxtoqec2fg9f0y",
+  "tenant_id": "5ceea899-ae8c-4ff1-fffe-353646eeeff0"
+}
+```
+
+## Technical Details
+
+### Actions
+
+#### List Watchlists
+
+This action is used to list watchlists.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|resourceGroupName|string|None|True|The name of the resource group within the user's subscription. The name is case insensitive|None|resourcegroup1|
+|subscriptionId|string|None|True|Azure subscription ID|None|d0cfe6b2-9ac0-4464-9919-dccaee2e48c0|
+|workspaceName|string|None|True|The name of the workspace|None|workspace1|
+
+Example input:
+
+```
+{
+  "resourceGroupName": "resourcegroup1",
+  "subscriptionId": "d0cfe6b2-9ac0-4464-9919-dccaee2e48c0",
+  "workspaceName": "workspace1"
+}
+```
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|watchlists|[]Watchlist|False|List of watchlists|
+
+Example output:
+
+```
+{
+  "watchlists":[
+    {
+      "id":"/subscriptions/0cea5e5b-a751-42ca-9151-dg965e27aefe/resourceGroups/integrationLab/providers/Microsoft.OperationalInsights/workspaces/sentinel/providers/Microsoft.SecurityInsights/Watchlists/Od nendzy do piniendzy",
+      "name":"Od nendzy do piniendzy",
+      "type":"Microsoft.SecurityInsights/Watchlists",
+      "systemData":{
+        "createdAt":"2022-04-12T18:37:46.6066473Z",
+        "createdBy":"cbebe4c0-d1a1-43e8-973a-5292f659c22e",
+        "createdByType":"Application",
+        "lastModifiedAt":"2022-04-12T18:37:46.6066473Z",
+        "lastModifiedBy":"cbebe4c0-d1a1-43e8-973a-5292f659c22e",
+        "lastModifiedByType":"Application"
+      },
+      "properties":{
+        "watchlistId":"8964710e-13fb-4b57-a6b6-90592a92b0fb",
+        "displayName":"High Value Assets Watchlist",
+        "provider":"Microsoft",
+        "source":"Local File",
+        "itemsSearchKey":"header1",
+        "created":"2022-04-12T18:37:46.6066473+00:00",
+        "updated":"2022-04-12T18:37:46.6066473+00:00",
+        "createdBy":{
+          "objectId":"dd702e59-e27b-48f6-95df-05c12f3f47c4",
+          "name":"cbebe4c0-d1a1-43e8-973a-5292f659c22e"
+        },
+        "updatedBy":{
+          "objectId":"dd702e59-ed7d-48d6-15d1-15a12a3f47e4",
+          "name":"cbebe4c0-d1a1-43e8-973a-5292f659c22e"
+        },
+        "description":"Watchlist from CSV content",
+        "watchlistType":"watchlist",
+        "watchlistAlias":"Od nendzy do piniendzy",
```

Comment:
Could you please replace the watchlist name with "Example watchlist" or some other English term? 😅